components/reusable-components/footer/footer.module.scss

```diff
@@ -0,0 +1,142 @@
+@mixin respond-to($size) {
+  @if $size == 'small' {
+    @media (max-width: 768px) {
+      @content;
+    }
+  } @else if $size == 'medium' {
+    @media (min-width: 769px) and (max-width: 1024px) {
+      @content;
+    }
+  } @else if $size == 'large' {
+    @media (min-width: 1025px) and (max-width: 1440px) {
+      @content;
+    }
+  } @else if $size == 'extra-large' {
+    @media (min-width: 1441px) {
+      @content;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  background-color: var(--bg-color);
+  color: var(--text-color);
+}
+
+.footerLogo {
+  flex: 1;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  padding-left: 1.5em;
+  padding-bottom: 6.5em;
+}
+
+.contactSection {
+  padding-bottom: 8.18em;
+}
+
+.contactSection p:first-child {
+  flex: 2;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  margin-right: 5em;
+  margin-left: 16em;
+  margin-bottom: 1.3em;
+}
+
+.contactSection a {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  margin-right: 6.25em;
+  padding-left: 16em;
+  padding-bottom: 8em;
+  text-decoration: underline;
+  color: inherit;
+}
+
+.socialMediaSection {
+  flex: 2;
+  font-family: 'Roboto Condensed', sans-serif;
+  font-weight: 700;
+  text-transform: uppercase;
+  padding-bottom: 2.5em;
+  padding-left: 4em;
+}
+
+.socialMediaSection p {
```

Comment:
For better understanding and code improvement, we can nest the `p` and `a` style tags into the class `.socialMediaSection` as a parent class.

Reply:
@Mikivitkovski that can be applied for other sections as well.